src/applications/personalization/profile360/selectors.js

```diff
@@ -28,13 +28,5 @@ export const directDepositAddressIsSetUp = state => {
 export const directDepositIsBlocked = state => {
   const controlInfo =
     directDepositInformation(state)?.responses?.[0]?.controlInformation || {};
-  // If all of these flags are not strictly equal to `true` we want to block
-  // access to the direct deposit feature. ie, we want to err on the side of
-  // blocking people from accessing direct deposit rather than err on the side
-  // of letting people have access to the feature.
-  return (
-    controlInfo.isCompetentIndicator !== true ||
-    controlInfo.noFiduciaryAssignedIndicator !== true ||
-    controlInfo.notDeceasedIndicator !== true
-  );
+  return controlInfo.canUpdateAddress !== true;
```

Comment:
This logic has been greatly simplified. We already get a `canUpdateAddress` "superflag" that will only be `true` if all the other `controlInformation` flags are `true` so there is no reason to check the values of these three flags. The plan is to _not_ treat different reasons differently so we can just do this simple check.